src/routes/adverse.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { getAllAdverseReport, AddAdverse } from '../controller/adverse';
+
+
+const router = express.Router();
+
+router.get('/adverse', getAllAdverseReport);
```

Comment:
It's better to have the`'/adverse'` part of the route to app.ts file like `app.use('/adverse', AdverseRoute).`
So that you won't have to repeat it in every route file. Follow this approach for all the routes